make refresh broken in Docker Desktop 3.5.1

Docker desktop just updated to Docker version 20.10.7, build f0df350 and `make refresh` doesn't work properly anymore, when I run `make refresh` I get

```sh
jaredlockhart@mozilla-jkerim~/Documents/experimenter(5739)$ make refresh&&make up
docker-compose -f docker-compose.yml kill
[+] Running 0/0
 ⠿ Container experimenter_redis_1               Error while Killing                                                                                                             0.0s
 ⠿ Container experimenter_db_1                  Error while Killing                                                                                                             0.0s
 ⠿ Container experimenter_kinto_1               Error while Killing                                                                                                             0.0s
 ⠿ Container experimenter_app_run_f23d13e98ce3  Error while Killing                                                                                                             0.0s
 ⠿ Container experimenter_autograph_1           Error while Killing                                                                                                             0.0s
Error response from daemon: Cannot kill container: 63ca282cab4ac9c5217b91f3af41014f45af6e9835d6db7cd0ce8fd1904c77ea: Container 63ca282cab4ac9c5217b91f3af41014f45af6e9835d6db7cd0ce8fd1904c77ea is not running
make: *** [compose_stop] Error 1
```

So let's update `make refresh` to pass silently over this.

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/EXP-1469)

--- a/Makefile
+++ b/Makefile
@@ -70,14 +70,14 @@ compose_build: build_dev ssl
 	$(COMPOSE)  build
 
 compose_stop:
-	$(COMPOSE) kill
-	$(COMPOSE_INTEGRATION) kill
-	$(COMPOSE_PROD) kill
+	$(COMPOSE) kill || true
+	$(COMPOSE_INTEGRATION) kill || true
+	$(COMPOSE_PROD) kill || true
 
 compose_rm:
-	$(COMPOSE) rm -f -v
-	$(COMPOSE_INTEGRATION) rm -f -v
-	$(COMPOSE_PROD) rm -f -v
+	$(COMPOSE) rm -f -v || true
+	$(COMPOSE_INTEGRATION) rm -f -v || true
+	$(COMPOSE_PROD) rm -f -v || true
 
 volumes_rm:
 	docker volume prune -f

